src/components/card-container/card-container.js

```diff
@@ -51,7 +46,7 @@ CardContainer.propTypes = {
   path: PropTypes.string,
   fullWidthCards: PropTypes.bool,
   cards: PropTypes.arrayOf(PropTypes.node).isRequired,
-  cardColSize: PropTypes.oneOf([1, 2, 3, 4, 5, 6])
+  cardColSize: PropTypes.oneOf(['1/4', '1/3', '1/2'])
```

Comment:
What do you think about limited the `cardColSize` options? I don't think any of the (soon-to-be) deprecated sizes are in use.

Reply:
I think the limited options make sense! I do see cardColSize `5` used here https://github.com/mapbox/help/blob/publisher-production/src/pages/tutorials/create-a-map-with-data-visualization-component.md#choose-a-visualization-type, but it seems that we could change this to `1/2` 🤔 as well as https://github.com/mapbox/help/blob/publisher-production/src/pages/getting-started/add-markers.md#use-mapbox-gl-js

Reply:
Thanks for catching that! I agree seems like `1/2` would be ideal there!